This cmake file uses command CHECK_LIBRARY_EXISTS that is defined in
module CheckLibraryExists, but it does not include that module.

This caused build error in lldb, but only on x86 (not x86_64)...

  -- Performing Test HAVE_CXX_ATOMICS64_WITHOUT_LIB - Failed
  CMake Error at /usr/lib/llvm4/lib/cmake/llvm/CheckAtomic.cmake:66 (check_library_exists):
    Unknown CMake command "check_library_exists".
  Call Stack (most recent call first):
    cmake/modules/LLDBStandalone.cmake:90 (include)
    CMakeLists.txt:3 (include)

--- a/cmake/modules/CheckAtomic.cmake
+++ b/cmake/modules/CheckAtomic.cmake
@@ -1,6 +1,7 @@
 # atomic builtins are required for threading support.
 
 INCLUDE(CheckCXXSourceCompiles)
+INCLUDE(CheckLibraryExists)
 
 # Sometimes linking against libatomic is required for atomic ops, if
 # the platform doesn't support lock-free atomics.
